<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="messageTextBrowser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>401</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="userTableWidget">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>40</y>
     <width>271</width>
     <height>491</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ip</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>端口</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="userNumberLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>550</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户在线：0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
